packages/oui/src/utils/createMediaPreload.js

```diff
@@ -1,6 +1,6 @@
 import * as React from 'react'
 
-const SINGLE_SRC_MEDIA_TYPES = ['image', 'video', 'audio', 'iframe']
```

Comment:
How come the `iframe` value was removed? Because of the reliance of `as="document"` on the `<link/>` element?

Reply:
Actually I removed it as I do not know, and never tested if iframes can be preloaded via a link tag.

Reply:
Ah, cool!